### PR TITLE
Create africhi.csl

### PR DESCRIPTION
**Pull request by andrew-from-pretoria**
For the AfriCHI conference

### CODE REVIEW

**Comment by andrew-from-pretoria** [comment]
seems like the file name can't have digits.

**Comment by adam3smith** [comment]
digits are fine, uppercase letters aren't. The filename should be africhi-2016.csl if you want the title as AfriCHI 2016 or just africhi.csl if you want to leave it as is. 
See https://github.com/citation-style-language/styles/wiki/Style-Requirements for more.

**Comment by adam3smith** [comment]
let us know if you need any help.

**Comment by andrew-from-pretoria** [comment]
Licensing wording is an issue

**Comment by andrew-from-pretoria** [comment]
Please help to correctly give licensing information. The information is in the file but the way it is put seems to be incorrect. Thanks

The current csl file has:

<rights>This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/</rights>

The error message is:
"  1) africhi: must have the correct Creative Commons BY-SA license URL
     Failure/Error: expect(style.info.rights[:license]).to eq(CSL_LICENSE_URL)
     
       expected: "http://creativecommons.org/licenses/by-sa/3.0/"
            got: nil
     
       (compared using ==)
     Shared Example Group: "style" called from ./spec/styles_spec.rb:156
     # ./spec/styles_spec.rb:140:in `block (2 levels) in <top (required)>'
                                                                                
  2) africhi: must have the correct Creative Commons BY-SA license text
     Failure/Error: expect(style.info.rights.text).to eq(CSL_LICENSE_TEXT)
     
       expected: "This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License"
            got: "This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/"
     
       (compared using ==)"

File: africhi.csl [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    </author>
+    <contributor>
+      <name>Chris Horn</name>
+      <email>chris.horn@securedecisions.com</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="engineering"/>
+    <updated>2016-07-20T09:53:16+00:00</updated>
+    <rights>This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/</rights>
```
Comment (adam3smith) [inline africhi.csl]: this line should be:
` <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>`

File: africhi.csl [hunk 2]
```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>africhi</title>
```
Comment (adam3smith) [inline africhi.csl]: The title (and only the title) should still have caps as expected by the user, i.e. AfriCHI

File: africhi.csl [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    <id>http://www.zotero.org/styles/africhi</id>
+    <link href="http://www.zotero.org/styles/africhi" rel="self"/>
+    <link href="http://www.acm.org/sigs/publications/proceedings-templates" rel="documentation"/>
```
Comment (adam3smith) [inline africhi.csl]: we'd want the documentation for the actual style here. This is still SIG-CHI

File: africhi.csl [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    <author>
+      <name>Naeem Esfahani</name>
+      <email>nesfaha2@gmu.edu</email>
+      <uri>http://mason.gmu.edu/~nesfaha2/</uri>
+    </author>
```
Comment (rmzelle) [inline africhi.csl]: Per https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link, we prefer it if you delete any authors/contributors from the original style, and add a "template" link instead.

Comment (andrew-from-pretoria) [inline africhi.csl]: "template" link? Please help, I have no idea what that is. Thank you

Comment (rmzelle) [inline africhi.csl]: Did you see the description at the link above?

I'm not a 100% sure which ACM SIG CSL style you based africhi.csl on. But, for example, if you based this style off of https://github.com/citation-style-language/styles/blob/master/acm-sig-proceedings.csl, the "template" link would be 

```xml
<link href="http://www.zotero.org/styles/acm-sig-proceedings" rel="template"/>
```

**Comment by rmzelle** [comment]
Closing (for now) because of inactivity. @andrew-from-pretoria, if you get back to us and address our requests, we would be happy to reopen this pull request and help you further to get this style submitted.

**Comment by andrew-from-pretoria** [comment]
Understood

Thank you for your assistance thus far.

Kind regards
Andrew


>>> On 14/08/2016 at 04:46, in message
<citation-style-language/styles/pull/2151/c239652814@github.com>, "Rintze M.
Zelle" <notifications@github.com> wrote:
- Closing (for now) because of inactivity. @andrew-from-pretoria, if you get 
- back to us and address our requests, we would be happy to reopen this pull 
- request and help you further to get this style submitted.
- 
- -- 
- You are receiving this because you were mentioned.
- Reply to this email directly or view it on GitHub:
- https://github.com/citation-style-language/styles/pull/2151#issuecomment-239 
- 652814


--

This message is subject to the CSIR's copyright terms and conditions, e-mail legal notice, and implemented Open Document Format (ODF) standard. 
The full disclaimer details can be found at http://www.csir.co.za/disclaimer.html. 

Please consider the environment before printing this email.

**Pull request closed without merging**